src/bioregistry/data/bioregistry.json

```diff
@@ -16761,6 +16761,25 @@
       "uri_format": "http://identifiers.org/fungidb/$1"
     }
   },
+  "fyeco": {
+    "banana": "FYECO",
```

Comment:
@matentzn not sure if this should even be an option at this point - the redundant namespace thing is just a way of fixing the goofy choices from Identifiers.org

Reply:
A banana is the literal string that shows as a redundant version of the prefix sometimes in an local unique identifier.